src/cloudflare/internal/images-api.ts

```diff
@@ -303,6 +327,165 @@ function chainStreams<T>(streams: ReadableStream<T>[]): ReadableStream<T> {
   return outputStream;
 }
 
+function concatUint8Arrays(a: Uint8Array, b: Uint8Array): Uint8Array {
```

Comment:
This function doesn't take care of different offsets. Also we can use resizable array buffers, as far as I know, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/resize

Reply:
I think it does work with offsets: https://www.typescriptlang.org/play/?#code/GYVwdgxgLglg9mABBBECGUCqMxQBwCCATkWgJ4DOAFGgFyLa6EnkA0iARvY-saWQEpuOXizKIA3gChEyBBSiIiAUwogANooC8iMMoDuDEc340AdOuVgA5lAAWiANScLV23YEBuGUtUaoZhTKUDTsAAxePipqmoHBVBzsaK429pGyKlAgREjR-t4AvlJSKGAKiGiIOjwm5GbARHAAtlQA2gCM7ABM7ADM7AAs7ACsALqRpeUcVUZMfHUNzW0AbOwA7Ox47ACc7O1h494l8nCWFnDWVKXoWMbzlKGcAhMnZ+oXV6gYNffUaOwcQIgDhoMRULrPLxAA

Holding off on the resizable bit as this would be a more sizeable change and it would be much better to use `Buffer` instead